pytests/tests/test_pyfunctions.py

```diff
@@ -87,18 +88,18 @@ def test_args_kwargs_rs(benchmark):
     assert rust == py
 
 
-def positional_only_py(a, /, b):
-    return a, b
+if sys.version_info >= (3, 8):
 
+    def positional_only_py(a, /, b):
```

Comment:
still fails: https://github.com/PyO3/pyo3/actions/runs/14672375510/job/41181694740#step:28:993

I think SyntaxErrors are caught by the compiler, so you can't avoid hitting it with an if statement like this. Is there a different way to spell this that doesn't require actually using the `, /,` syntax?

Reply:
Python is not as dynamic as I had hoped. Fixed by removing the `, /` (a bit less nice but likely not a big deal